dart/test/sentry_client_test.dart

```diff
@@ -551,3 +602,17 @@ SentryEvent beforeSendCallback(SentryEvent event, {dynamic hint}) {
 SentryEvent? eventProcessorDropEvent(SentryEvent event, {dynamic hint}) {
   return null;
 }
+
+/// Test Fixture for tests with [SentryOptions.sendDefaultPii]
+class PiiFixture {
+  PiiFixture(bool sendDefaultPii) {
+    var options = SentryOptions(dsn: fakeDsn);
+    options.sendDefaultPii = sendDefaultPii;
+    transport = MockTransport();
+    options.transport = transport;
+    client = SentryClient(options);
+  }
+
+  late MockTransport transport;
+  late SentryClient client;
+}
```

Comment:
```suggestion
class Fixture {
  Fixture(bool sendDefaultPii) {
    var options = SentryOptions(dsn: fakeDsn);
    options.sendDefaultPii = sendDefaultPii;
    transport = MockTransport();
    options.transport = transport;
    client = SentryClient(options);
  }
  
  SentryClient getSut(bool sendDefaultPii = true) {
    var options = SentryOptions(dsn: fakeDsn);
    transport = MockTransport();
    options.transport = transport;
    return SentryClient(options);
  }

  late MockTransport transport;
  late SentryClient client;
}
```

Reply:
thats pseudocode, but just to give a hint on how it could be, having a generic name helps for copypasta to other tests, also, getSut() creates a new instance of SentryClient so on each tests, you have a clean state, you don't actually need to create a Fixture per test, but a Sut per test

Reply:
Okay, I hope I got it right now :D